AEPPlaces/Sources/NamedCollectionDataStore+Places.swift

```diff
@@ -65,6 +65,18 @@ extension NamedCollectionDataStore {
     var lastKnownLongitude: Double {
         return getDouble(key: PlacesConstants.UserDefaults.PERSISTED_LONGITUDE) ?? PlacesConstants.DefaultValues.INVALID_LAT_LON
     }
+    
+    var accuracy: CLAccuracyAuthorization? {
+        guard #available(iOS 14, *) else {
+            return nil
+        }
+        
+        if let persistedAccuracy = getString(key: PlacesConstants.UserDefaults.PERSISTED_ACCURACY) {
+            return CLAccuracyAuthorization(fromString: persistedAccuracy)
+        }
+
+        return nil
+    }
```

Comment:
curious if the enum `CLAccuracyAuthorization` available only after iOS 14

should we wrap the complete accuracy property with #available(iOS 14, *)
I do see this being compile properly on iOS 13 devices.. But wonder how